scripts/run-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+function checkForConditionalRun {
+  if [ "$TRAVIS" == "ci" ];
+  then
+    echo "Checking if this should be conditionally run.."
+    result=$(git diff --name-only HEAD | grep azurerm/)
+    if [ "$result" = "" ];
+    then
+      echo "No changes committed to ./azurerm - nothing to lint - exiting"
+      exit 0
+    fi
+  fi
+}
+
+function runTests {
+  echo "==> Running Unit Tests..."
+  go test -i $TEST || exit 1
```

Comment:
globbing / splitting protection
```suggestion
  go test -i "$TEST" || exit 1
```

Reply:
in testing, apparently not? ¯\_(ツ)_/¯